callback_plugins/default_plus_summary.py

```diff
@@ -0,0 +1,94 @@
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible callback plugin.
+'''
+
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible import constants as C
+from ansible.utils.color import stringc
+
+
+class CallbackModule(CallbackModule_default):
+    '''
+    This is like the default callback plugin, but also stores results and
+    summarizes failures.
+    '''
+
+    CALLBACK_VERSION = 2.0
+    CALLBACK_TYPE = 'stdout'
```

Comment:
Nevermind, I got it working.

I can't find much documentation around plugins... my best friend has been the ansible source code.

It seems that [apart from `stdout` and `tree`, the ansible code ignores the CALLBACK_TYPE completely](https://github.com/ansible/ansible/blob/ad1e0665401dd436b2043ed64b8ffcad49e205b7/lib/ansible/executor/task_queue_manager.py#L185-L190)... it seems to be of just an informational character, and maybe a gentlemen's agreement not written down anywhere?!

FWIW, setting `CALLBACK_TYPE = 'lorem-ipsum'` produces the same result as `'aggregate'` :smile: